projects/exui-common-lib/src/lib/components/write-address/write-address.component.ts

```diff
@@ -195,4 +196,9 @@ export class WriteAddressFieldComponent implements OnInit, OnChanges {
       );
     }
   }
+
+  private setPostcodeForm(): void {
+    const postcodeNeeded = this.isInternational === undefined && this.formGroup.get('address') && this.formGroup.get('address').get('postCode');
```

Comment:
Can be replaced by 
`const postcodeNeeded = !this.isInternational && this.formGroup.get('address') && this.formGroup.get('address').get('postCode');`